languages/js/tsconfig.json

```diff
@@ -12,9 +12,10 @@
     "rootDir": ".",
     "experimentalDecorators": true,
     "emitDecoratorMetadata": true,
+    "strictPropertyInitialization": false
```

Comment:
This seems not great; what's the case where we can't set a sensible default in the constructor and then update it sometime thereafter?

Reply:
fixed!